classtime.js

```diff
@@ -1,209 +1,169 @@
-var currentDay = 0;
-var currentHours = 0;
-var currentMinutes = 0; 
-var currentSeconds = 0;
+let DAY_OFF_FLAG = "day off"
```

Comment:
I'd recommend putting the flags into an object/enum to really drive home the fact that they're flags (vs. constant variables/strings). This way, anytime you reference a flag it would show as `ScheduleFlags.DAY_OFF_FLAG`.

Reply:
yeah that sounds good. one of us can add it when we have time

Reply:
this has been done (i think) in the react version